### PR TITLE
[10.0] Port partner_address_street3

### PR DESCRIPTION
**Pull request by alexis-via**
Add %(street3)s in 'address_format' of existing countries

### CODE REVIEW

**Comment by alexis-via** [comment]
Linked to #309

**Comment by atchuthan** [comment]
code review :+1:

**Comment by atchuthan** [comment]
@alexis-via there were few corrections in hooks.py for this module in v9.0, please update it for 10.0

**Comment by alexis-via** [comment]
It's ready to merge now.

File: partner_address_street3/tests/test_street_3.py [hunk 1]
```diff
@@ -1,88 +1,36 @@
...
-            self.uid,
-            bart_id,
-        )
+            'type': 'contact',
+            })
+        self.assertEquals(bart.street3, 'Tho')
 
-        self.assertTrue(bart.street3, 'Tho')
+        # test synchro of street3 on write
+        homer.write({'street3': 'in OCA we trust'})
```
Comment (rvalyi) [inline partner_address_street3/tests/test_street_3.py:35]: :-)

**Review by atchuthan — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**